<a href="https://colab.research.google.com/github/bismahashim/Assignment-2/blob/main/Assignment_Set_2__Q5.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm

In [3]:
# Mean of both divisions
mean = 5+7
mean

12

In [5]:
# STD of both division
sd = np.sqrt((9)+(16))
sd

5.0

In [10]:
# condition $1 = Rs 44 , in million hence
Mean = mean*45
SD = sd*45 
Mean

540

In [11]:
SD

225.0

In [12]:
#A.	Specify a Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.
stats.norm.interval(0.95,540,225)


(99.00810347848784, 980.9918965215122)

Hence Mean = Rs 540 in Millions
SD = Rs 225.0 in Millions
Range = Rs (99.00810347848784, 980.9918965215122) in Millions 

In [13]:
#To compute 5th Percentile, we use the formula X=μ + Zσ; wherein from z table, 5 percentile = -1.645
X= 540+(-1.645)*(225)
X

169.875

In [14]:
#C.	Which of the two divisions has a larger probability of making a loss in a given year?
# Probability of Division 1 making a loss P(X<0)
stats.norm.cdf(0,5,3)

0.0477903522728147

In [15]:
# Probability of Division 2 making a loss P(X<0)
stats.norm.cdf(0,7,4)

0.040059156863817086